### DISCUSSION TITLE
Renaming Docs Menu

### DISCUSSION THREAD

**Discussion by ampacheco** [thuliteio/doks#419] [Q&A] [answered]
Renaming Docs Menu in [menus.toml](/h-enk/doks/blob/master/config/_default/menus.toml) does not show the menu selected as expected.

I was trying to solve issue by may self but I an new to Hugo/doks and I do not understand well layouts. I see that we have a layout under root folder and other with far more information under node_modules/@hyas/doks/layouts. I figure it out that the second one is a template but I am not sure the relation between them and where to act.

Thanks,
Alejandro

**Comment by abusayed0206** [comment]
Facing the same issue!
Need a detailed guideline to make like this : 
![image](https://user-images.githubusercontent.com/43584104/129595956-5a78fefa-58e1-4ab3-beaa-0120f1e97276.png)

**Comment by h-enk** [comment, marked as answer]
> I see that we have a layout under root folder and other with far more information under node_modules/@hyas/doks/layouts. I figure it out that the second one is a template but I am not sure the relation between them and where to act.

@ampacheco, please see https://github.com/h-enk/doks/discussions/320#discussioncomment-868328

@abusayed0206, so you need to edit `./config/_default/menus.toml` like so: [Add to docs menu](https://getdoks.org/tutorial/add-menus/#add-to-docs-menu)

To enable collapsible documentation navigation, in `./config/_default/params.toml`, set `collapsibleSidebar = true`.

**Reply by ampacheco** [reply]
Thank you @h-enk . Your answer open more question. I know that is for my lacks of knowledge in all tools & platforms involved and the way you abstract them for a great result. I am no sure that should be the right thread but I will try:

1. Even I change the name of docks in ./config/defaults/menus.toml is still and issue, after rename the menu never goes again as selected
2. When I use child as template, the once you recommend, for little or not customization at all docs menus appears sorted alphabetically even I used weight to control order, no way to manage the order I want
3. The last one is I am planning to publish in github pages, for both child and start does the process is the same? When I clone your [repo](https://github.com/h-enk/doks-gh-pages/tree/gh-pages) that support that everything works fine, when I try to do it by myself, I get a mesh. But I do not want start that way because version is too old 0.1

Thank you for the time you commit to the support.
